package.json

```diff
@@ -26,14 +26,16 @@
     "@wdio/cli": "^7.11.1",
     "@wdio/local-runner": "^7.11.1",
     "@wdio/mocha-framework": "^7.11.1",
+    "@wdio/selenium-standalone-service": "^7.10.1",
     "@wdio/spec-reporter": "^7.10.1",
+    "allure-commandline": "^2.13.8",
     "chromedriver": "^92.0.2",
-    "geckodriver": "^2.0.4",
+    "module": "^1.2.5",
     "prettier": "^2.3.2",
+    "set-blocking": "^2.0.0",
     "wdio-chromedriver-service": "^7.2.0"
   },
   "dependencies": {
-    "allure-commandline": "^2.13.8",
-    "set-blocking": "^2.0.0"
+    "chai": "^4.3.4"
   }
 }
```

Comment:
I'd suggest you to discard all the installed dependencies by running 'git checkout main package.json', then install chai dependency back and push the changes again. This way you'll have only the necessary changes on this branch.

Reply:
Can you be more specific about the suggestion please? 
'git checkout main package.json'  gives "Updated 0 paths from 45d8abd" message.